models/m_payment.erl

```diff
@@ -452,14 +506,53 @@ install(Context) ->
                 create index payment_psp_external_id_key
                 on payment (psp_external_id)",
                 Context),
+            [] = z_db:q("
+                create index fki_payment_recurring_payment_id
+                on payment (recurring_payment_id)",
+                Context),
+            [] = z_db:q("
+                ALTER TABLE payment
+                ADD CONSTRAINT fk_payment_recurring_payment_id FOREIGN KEY (recurring_payment_id)
+                REFERENCES payment (id)
+                ON UPDATE CASCADE ON DELETE RESTRICT",
```

Comment:
This means that a recurring payment mandate providing payment cannot be deleted if automatic payments exist referencing it.

Reply:
Isn't it better to use `add_recurring_payment_id_column` here, instead of duplicating code?